spaceship/lib/spaceship/connect_api/models/bundle_id_capability.rb

```diff
@@ -59,14 +74,26 @@ def is_type?(type)
       # API
       #
 
-      def self.all(client: nil, filter: {}, includes: nil, limit: nil, sort: nil)
+      def self.all(client: nil, bundle_id_id: nil, filter: {}, includes: nil, limit: nil, sort: nil)
+        raise "bundle_id_id is required " if bundle_id_id.nil?
```

Comment:
FYI: You can use `ArgumentError` for this kind of validation in Ruby in general.


```suggestion
        raise ArgumentError, 'bundle_id_id is required' if bundle_id_id.nil?
```

(You don't have to check if `bundle_id_id` is given manually since you can remove default value)